README.md

```diff
@@ -145,7 +202,7 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
+I wrote mine in java. As long as JVM is intalled, all that required is that while in this directory, in a terminal type 'java Sentance' and it shall run perfectly with 3 correct and 3 incorrect test cases.
```

Comment:
And run perfectly it did.

I appreciate you adding passing and failing tests, and that you checked them automatically.

However, your state machine is not table based: you've hard coded it (quite elegantly) as a set of methods.

I was really looking for something that actually had the table in it ("Translate this table into a programming language of your choice"). However, I like your method-based approach, so I'm dinging you less than I would otherwise.

# -3